routes.go

```diff
@@ -93,6 +94,8 @@ func InitRoutes(apper Apper, r *mux.Router) *mux.Router {
 	me.HandleFunc("/posts/export.json", handler.Download(viewExportPosts, UserLevelUser)).Methods("GET")
 	me.HandleFunc("/export", handler.User(viewExportOptions)).Methods("GET")
 	me.HandleFunc("/export.json", handler.Download(viewExportFull, UserLevelUser)).Methods("GET")
+	me.HandleFunc("/import", handler.User(viewImport)).Methods("GET")
+	me.HandleFunc("/import", handler.User(handleImport)).Methods("POST")
```

Comment:
In general, we should try to keep endpoints that perform actions on the `/api/` subroute. So I think this should be on `/api/me/import` instead of `/me/import`